std/testing/asserts_test.ts

```diff
@@ -409,3 +409,102 @@ Deno.test({
     );
   },
 });
+
+Deno.test("Assert Throws Non-Error Fail", () => {
+  assertThrows(
+    () => {
+      throw new Error("Panic!");
+    },
+    Error,
+    "Panic!"
+  );
```

Comment:
Certainly this case has already been covered by previous tests. I think it should be removed. Can you shorten the test to only that which is being fixed?

Reply:
Yep, sure. 👍